pkg/controller/render/render_controller.go

```diff
@@ -603,6 +608,32 @@ func generateRenderedMachineConfig(pool *mcfgv1.MachineConfigPool, configs []*mc
 	return merged, nil
 }
 
+// This function takes into account the current MachineConfig that is present on
+// the MachineConfigPool. It first attempts to reconcile each individual
+// MachineConfig against the current rendered MachineConfig, then it generates
+// the new rendered MachineConfig, before finally verifying that the complete
+// rendered MachineConfig can be reconciled. We make two reconciliation passes
+// because we want to identify any MachineConfigs that fail the reconcilable
+// check. We verify reconiliation of the generated MachineConfig because we
+// want to take the ControllerConfig into consideration (although perhaps this
```

Comment:
Note to reviewer: Keep me honest with this statement. Is it possible that changes from the ControllerConfig could not be reconcilable?

Reply:
I don't believe so. ControllerConfig changes should only ever touch files/units and kargs(?)